{"name": "test_car", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: assert {'name': 'Truck'} == 'Truck'", "trace": "car = <function car.<locals>._car at 0x7ff318dc1790>\n\n    @pytest.mark.xfail\n    def test_car(car):\n        c1 = car(name=\"Truck\")\n        c2 = car(name=\"Bus\")\n    \n>       assert c1 == \"Truck\"\nE       AssertionError: assert {'name': 'Truck'} == 'Truck'\n\npytest_notebooks/tests/test_fixture.py:42: AssertionError"}, "start": 1662956141810, "stop": 1662956141810, "uuid": "68a18223-847d-43c9-8e93-6540a26e0ed8", "historyId": "40dcd7d75f6f0e5551bfbe6d41ee007d", "testCaseId": "51947874fcd511b4f8577bcb69902da0", "fullName": "pytest_notebooks.tests.test_fixture#test_car", "labels": [{"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "parentSuite", "value": "pytest_notebooks.tests"}, {"name": "suite", "value": "test_fixture"}, {"name": "host", "value": "XR-7s-MBP"}, {"name": "thread", "value": "14853-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_notebooks.tests.test_fixture"}]}